content/en/_index.html

```diff
@@ -8,7 +8,7 @@
 
 {{< blocks/cover>}}
 <div class="mx-auto">
-	<img src="/img/Kube-CF-white.png" width="300"><br><br>
```

Comment:
don't we want the white one at the front? I'm actually trying to remove the one in the top on the main page so we have just one visible

Reply:
let's leave it white. I am not a web designer and as user both are not that pretty anyway.